airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -192,6 +192,7 @@ def __init__(self, service, project_id):
     def run_query(
             self, bql, destination_dataset_table = False,
             write_disposition = 'WRITE_EMPTY',
+            create_disposition='CREATE_IF_NEEDED',
```

Comment:
That is a good point. I was thinking the same thing as well. Alright, I will put the new parameter to the backwards. Thanks for the comment.